src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/BOM/MaterialForm/MaterialForm.tsx

```diff
@@ -143,12 +145,34 @@ const MaterialForm: React.FC<MaterialFormProps> = ({ submitText, onSubmit, defau
     }
   };
 
+  const mockUnits: Unit[] = [
+    { name: 'kg', materials: [] },
+    { name: 'Pack of 10', materials: [] },
+    { name: 'Pack of 20', materials: [] },
+    { name: 'Pack of 30', materials: [] },
+    { name: 'meters', materials: [] }
+  ];
+
+  const adjustedUnits = mockUnits
+    .map((unit) => {
+      if (unit.name.startsWith('Pack of')) {
+        return { ...unit, name: 'Pack', materials: [] };
+      }
+      return unit;
+    })
+    .filter((unit, index, self) => index === self.findIndex((t) => t.name === unit.name));
+
+  const packSizes = mockUnits
+    .filter((unit) => unit.name.startsWith('Pack of'))
+    .map((unit) => unit.name.replace('Pack of ', ''));
```

Comment:
I think there will be a better way to do this, but i think we can change it once we understand what exactly were doing